pkg/crc/version/version.go

```diff
@@ -39,3 +58,40 @@ func GetCommitSha() string {
 func GetBundleVersion() string {
 	return bundleVersion
 }
+
+func getCRCLatestVersionFromMirror() (*semver.Version, error) {
+	var releaseInfo CrcReleaseInfo
+	client := &http.Client{
+		Timeout: 5 * time.Second,
+	}
+	response, err := client.Get(releaseInfoLink)
+	if err != nil {
+		return nil, err
+	}
+	defer response.Body.Close()
+	releaseMetaData, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return nil, err
+	}
+	err = json.Unmarshal(releaseMetaData, &releaseInfo)
```

Comment:
I still do not believe this is more concise than using a simple text file and a regex, as you still need boilerplate code to make this work.

I believe the double nature is lost here...

Reply:
@gbraad what you mean by `double nature is lost here` ? To unmarshal a json content that is always required right?